src/api/app/views/webui2/webui/main/_breadcrumb_items.html.haml

```diff
@@ -5,5 +5,5 @@
     = home_title
 - else
   %li.breadcrumb-item
-    %i.fas.fa-home.mr-1
-    = link_to home_title, root_path
+    = link_to root_path do
```

Comment:
I would add a title here, perhaps `'Homepage'` or `home_title`

Reply:
The whole point was to not add a title here. Though we could add a tooltip.

Reply:
Sorry I wasn't clear, I was referring to the `title` attribute of the `a` tag generated by `link_to`

Reply:
I'm fine with @bgeuken's approach. I would add the tooltip.